src/schema.ts

```diff
@@ -20,14 +21,15 @@ export const defaultNodes: NodeSerializer = {
       HeadingLevel.HEADING_5,
       HeadingLevel.HEADING_6,
     ][node.attrs.level - 1];
+    // TODO pass margin in Header node
```

Comment:
Does this need to be done before merging?

Reply:
no, this was fixed properly.  That was a work around until we Tyler helped add a margin at the end of an article.